### DISCUSSION TITLE
Главная | Node.js с примерами кода

### DISCUSSION THREAD

**Discussion by giscus** [xsltdev/nodejsdev.ru#29] [Announcements]
# Главная | Node.js с примерами кода

Node.js - это программная платформа, которая транслирует JavaScript в машинный код, исполняемый на стороне сервера

https://nodejsdev.ru/

**Comment by Izumra** [comment]
Информация о Node js  еще актуальна?

**Reply by abcwalk** [reply]
да

**Comment by kobakamladze** [comment]
когда будет nestjs? планируется ли добавление rabbitmq?

**Reply by bndby** [reply]
nest в очереди, rabbit пока не планируется
